crates/nu-cli/src/shell/filesystem_shell.rs

```diff
@@ -1099,3 +1114,44 @@ fn path_contains_hidden_folder(path: &PathBuf, folders: &[PathBuf]) -> bool {
     }
     false
 }
+
+fn convert_bytes_to_string_using_format(
+    bytes: u64,
+    format: Tagged<String>,
+) -> Result<UntaggedValue, ShellError> {
+    let byte = byte_unit::Byte::from_bytes(bytes as u128);
+    if byte.get_bytes() == 0u128 {
+        return Ok(UntaggedValue::string("—".to_string()));
+    }
+    match format.item().as_str() {
```

Comment:
One quick comment here. You could make `format.item().as_str()` lower case and then the patterns you try to match could only be "b", "kb", "mb", etc.

Reply:
fair enough, will change